lbrynet/core/Wallet.py

```diff
@@ -341,7 +342,8 @@ def _get_cached_claim(self, claim_id, check_expire=True):
                                                "WHERE claimId=?", (claim_id, ))
         response = None
         if r and claim_tx_info and r[0]:
-            rid, _, seq, claim_address, height, amount, supports, raw, chan_name, valid, ts = r[0]
+            rid, _, seq, claim_address, height, amount, supports, raw, chan_name, valid, ts = r[
+                0]
```

Comment:
This appears to be 98 chars, so it's less than 100. I would keep it as a single line. Looks weird with the `0]` all by its lonesome.